packages/truffle-contract-schema/.gitignore

```diff
@@ -1,4 +1,4 @@
 node_modules
 yarn.lock
 package-lock.json
-*.d.ts
\ No newline at end of file
+spec/index.d.ts
```

Comment:
(Same inquiry about nested .gitignores)

Reply:
Same here. This was making git ignore the added typings, so I set it to only ignore the output spec typings. (Looks like the pkg was setup to ignore the spec typings on purpose).